docs/metatype.dev/blog/2024-09-26-introducing-grpc-runtime/index.mdx

```diff
@@ -0,0 +1,81 @@
+import BlogIntro from "@site/src/components/BlogIntro";
+
+# Introducing gRPC Runtime
+
+<BlogIntro text="We're excited to announce the new gRPC Runtime feature in Metatype, further enhancing our platform's ability to create versatile and powerful backends through typegraphs." />
+
+## What is Metatype?
+
+Before we dive into the gRPC Runtime, let's recap what Metatype is all about:
+
+Metatype is a revolutionary platform that allows developers to create typegraphs - strongly-typed, composable backend structures. With Metatype, you can:
```

Comment:
nitpick: I'd drop the term revolutionary here. These kinds of phrasing have become stigmatized.